js/apps/system/_admin/aardvark/APP/frontend/js/views/documentsView.js

```diff
@@ -740,35 +764,55 @@
       var from = $('.modal-body #new-edge-from-attr').last().val();
       var to = $('.modal-body #new-edge-to').last().val();
       var key = $('.modal-body #new-edge-key-attr').last().val();
-
+      var body;
+      try { // TODO: refactor this.
```

Comment:
why does this need refactoring? if not anymore, then we should remove the comment.

Reply:
@hkernbach wanted this comment.